package-lock.json

```diff
@@ -4049,795 +4048,6 @@
       "resolved": "https://registry.npmjs.org/fs.realpath/-/fs.realpath-1.0.0.tgz",
       "integrity": "sha1-FQStJSMVjKpA20onh8sBQRmU6k8="
     },
-    "fsevents": {
```

Comment:
This was removed in my `` - A quick search provided this for context: https://github.com/npm/npm/issues/17722. It looks like npm 5.6 should prevent this thrash, I can try to update to that version and see if this continues to happen on future PRs? Not sure what version everyone else will be on though when updating / if this thrash is "okay" for now?

CC @arizzitano @jaebradley

Reply:
@fysheets after switching to this branch, blowing away `node_modules` and reinstalling from `lms-shell`, I was able to install without any errors.

Interestingly, I saw these warnings
![image](https://user-images.githubusercontent.com/8136030/35117154-be5130ba-fc5b-11e7-91f2-26c2b59907e7.png)

`paver update_assets` seemed to work as well!